sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -9,12 +9,72 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     this.listenTo(this.categories, 'sync change:unread_count', this.render);
   },
 
+  events: {
+    'click #login-submit': 'loginSubmit',
```

Comment:
No. Submit does a post and auto-refresh of the page so we'll never actually render the errors we catch in the JSON. We're overwriting the ability to do a submit by making it a click.